doc/api/crypto.md

```diff
@@ -2422,6 +2422,25 @@ const ciphers = crypto.getCiphers();
 console.log(ciphers); // ['aes-128-cbc', 'aes-128-ccm', ...]
 ```
 
+### `crypto.getCipherInfo(nameOrNid)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `nameOrNid`: {string|number} The name or nid of the cipher to query.
+* Returns: {Object}
+  * `name` {string} The name of the cipher
+  * `nid` {number} The nid of the cipher
+  * `blockSize` {number} The block size of the cipher.
```

Comment:
What does this return for stream ciphers, and for block ciphers that act as stream ciphers? I assume it will return the internal block size, which may or may not be meaningful. Either way, this should probably be documented.

Reply:
For all the stream ciphers, `blockSize` returns 1, let's actually omit this property for stream ciphers.